backend.native/compiler/ir/backend.native/src/org/jetbrains/kotlin/backend/konan/KonanConfig.kt

```diff
@@ -120,53 +109,9 @@ class KonanConfig(val project: Project, val configuration: CompilerConfiguration
     internal val includeBinaries: List<String> = 
         configuration.getList(KonanConfigKeys.INCLUDED_BINARY_FILES)
 
-    fun loadLibMetadata(): List<ModuleDescriptorImpl> {
-
-        val specifics = configuration.get(CommonConfigurationKeys.LANGUAGE_VERSION_SETTINGS)!!
-        val friendLibsSet = configuration.get(KonanConfigKeys.FRIEND_MODULES)?.map { File(it) }?.toSet()
-
-        val allMetadata = mutableListOf<ModuleDescriptorImpl>()
-        val friends = mutableListOf<ModuleDescriptorImpl>()
-
-        resolvedLibraries.forEach { library, packageAccessedHandler ->
-            profile("Loading ${library.libraryName}") {
-                // MutableModuleContext needs ModuleDescriptorImpl, rather than ModuleDescriptor.
-                val moduleDescriptor = DefaultKonanModuleDescriptorFactory.createModuleDescriptor(library, specifics, packageAccessedHandler)
-                allMetadata.add(moduleDescriptor)
-                friendLibsSet?.apply {
-                    if (contains(library.libraryFile)) friends.add(moduleDescriptor)
-                }
-            }
-        }
-
-        this.friends = friends.toSet()
-
-        return allMetadata
-    }
-
-    private var forwardDeclarationsModule: ModuleDescriptorImpl? = null
-
-    internal fun getOrCreateForwardDeclarationsModule(
-            builtIns: KotlinBuiltIns, storageManager: StorageManager? = null
-    ): ModuleDescriptorImpl {
-        forwardDeclarationsModule?.let { return it }
-        val result = createForwardDeclarationsModule(
-                builtIns,
-                storageManager ?: LockBasedStorageManager()
-        )
-
-        forwardDeclarationsModule = result
-        return result
-    }
-
-    internal val moduleDescriptors: List<ModuleDescriptorImpl> by lazy {
-        for (module in loadedDescriptors) {
-            // Yes, just to all of them.
-            module.setDependencies(loadedDescriptors + getOrCreateForwardDeclarationsModule(module.builtIns))
-        }
+    internal val specifics = configuration.get(CommonConfigurationKeys.LANGUAGE_VERSION_SETTINGS)!!
 
-        loadedDescriptors
-    }
+    internal val friendLibsSet = configuration.get(KonanConfigKeys.FRIEND_MODULES)?.map { File(it) }?.toSet()
```

Comment:
`friendModules`?

Reply:
Guess it was named like this before, but I agree that better naming here wouldn't hurt.

Reply:
OK, will fix